.github/workflows/create-release.yaml

```diff
@@ -4,9 +4,17 @@ env:
   IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/runtime-watcher-skr
   CODE_REPOSITORY: kyma-project/runtime-watcher
 on:
-  push:
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]-?*'
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "Release name"
+        default: ""
+        required: true
+      since:
+        description: "Changelog since"
```

Comment:
Do I understand this correctly, that now it's the operator (human) that provides this value manually? This is error prone. I think we should aim for generating changelog since the `last_release`, where `last_release` should be detected automatically to prevent human typing errors.